train.py

```diff
@@ -45,23 +45,25 @@ def generate_subspectrogram(duration_s=config.SUBSPECTROGRAM_DURATION_S, fft_rat
         checkpoint.restore(checkpoint_manager.latest_checkpoint)
         print(f"Restored checkpoint. Model {checkpoint.model.name} - epoch {checkpoint.epoch.value()}")
 
-    # for test we iterate over samples one by one
-    for epoch in range(checkpoint.epoch.value(), config.NB_EPOCHS):
+    loss_history = []
+    epoch_range = range(checkpoint.epoch.value(), config.NB_EPOCHS) 
 
-        print(f"============================ epoch {epoch} =============================")
+    # for test we iterate over samples one by one
+    for epoch in epoch_range:
 
         for iteration, (spectro, label) in enumerate(train_dataset):
             predictions, labels = train_step(spectro, label, model, optimizer, train_loss, train_accuracy)
 
-            # display metrics
-            if iteration % 10 == 0:
-                utils.display_and_reset_metrics(train_loss, train_accuracy, predictions, labels, iteration=iteration)
+        # display metrics
+        loss_history.append(train_loss.result())
+        utils.display_and_reset_metrics(train_loss, train_accuracy, predictions, labels, epoch=epoch)
```

Comment:
Am I right in thinking this is now displayed every epoch ?

Reply:
It is on purpose. I think it is enough to display it after every epoch